### ISSUE TITLE
Add "feels like" temp

### ISSUE THREAD

**Issue by AeliusSaionji** [Akylas/oss-weather#50]
Hello, 
I find that the "feels like" temperature is often more useful than the literal temperature. Would you be willing to incorporate this into the app?

Thanks

**Comment by farfromrefug** [comment]
@AeliusSaionji i agree i think i will add an option to show feels like instead of literal. Though i dont think min/max are ever given as feels like. Would that be ok with you?

**Comment by farfromrefug** [comment]
@0-5-0 yes i need to do that! Though i know not all providers return it

**Comment by farfromrefug** [comment]
@0-5-0 the setting is now implemented in the latest version

**Comment by 0-5-0** [comment]
Seems to be working great, thank you :)
